frontend/public/components/utils/dropdown.jsx

```diff
@@ -137,6 +137,11 @@ export class Dropdown extends DropdownMixin {
     this.state.favoriteKey = favoriteKey;
     this.state.bookmarks = bookmarks;
 
```

Comment:
I left this comment because this issue created a bunch of debugging problems for me.  I'd like to come back and resolve it later.

Reply:
I'm not sure if we want to change this behavior. For one, if you log in as different users, you have access to different projects.

Reply:
Right, the point would probably be to do a check & ensure the project still exists... otherwise clear the bookmark.  Or use some kind of identifier to store the bookmark for different users, etc.

Reply:
We can't assume a project doesn't exist or even that you can't see any resources in it just because it's not in the project list. We need to handle this case.